app/assets/javascripts/quotingPanel.js

```diff
@@ -85,13 +114,58 @@ class QuotingPanel {
   addListeners() {
     document.getElementById("SubmitQuote").addEventListener('submit', (event) => { event.preventDefault() });
     document.getElementById("searchAddressButton").addEventListener('click', this.handleSearchAddressClick);
+    document.getElementsByName("serviceExpeditionCost").forEach((element) => { 
+      element.addEventListener('click', (event) => { 
```

Comment:
As with all the other handlers in `addListeners, move the body of this to a `handleExpeditionCostClick` function and leave as:
```javascript
  document.getElementsByName("serviceExpeditionCost").forEach((element) => { 
    element.addEventListener('click', this.handleExpeditionCostClick);
  }
```

Reply:
done